client/src/App.tsx

```diff
@@ -24,6 +25,12 @@ function App(): JSX.Element {
                 <Route exact path="/dashboard">
                   <Dashboard />
                 </Route>
+                <Route exact path="/edit-profile">
+                  <EditProfile />
+                </Route>
+                <Route path="/edit-profile/:id">
```

Comment:
Why do we have two routes for `edit-profile`? Each user will only be associated with one profile, so we don't need two routes here